scripts/rollup/results.json

```diff
@@ -1,17 +1,17 @@
 {
-  "branch": "master",
+  "branch": "warnForKeysInFragmentsRefactor",
```

Comment:
tldr; In most cases no, there won't be terrible merge conflicts, and when it happens it should be fairly easy to resolve.

I think I have been in similar situations with generated 'schema' type files, and there can be annoying merge conflicts in some cases.

If people have a stack of commits, and they commit the 'results.json' in each commit, and then rebase, they could have a merge conflict to resolve for each diff.

Ideally they just commit the 'results.json' at the top of their commit stack, and if there is a conflict they can regenerate the file and `git add scripts/rollup/results.json`.

It would be great to automate this though.